### DISCUSSION TITLE
Facebook 集群调度管理系统 · OSDI 2020 - 面向信仰编程 · /papers-twine

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#635] [Announcements]
https://draveness.me/papers-twine/

**Comment by Sailfishc** [comment]
等了好久，原来在写书😝，2021爆肝4篇

**Comment by draveness** [comment]
> 等了好久，原来在写书😝，2021爆肝4篇

不容易啊

**Comment by yuchanns** [comment]
爷爷你关注的左神终于更新了！
顺便问一句为啥配图的风格变了

**Comment by draveness** [comment]
> 爷爷你关注的左神终于更新了！
> 顺便问一句为啥配图的风格变了

原来的风格看多了，想换个风格

**Comment by draveness** [comment]
> @Yanwenjiepy
> mesos也应该看下

Mesos 了解的不多
